articles/how-to-optimize-wordpress-website-speed/index.md

```diff
@@ -0,0 +1,161 @@
+
+It is recommended that you compress images before uploading them on WordPress. Photoshop, [compressor.io](https://compressor.io/) and [BulkResize](https://bulkresizephotos.com/en) are some of the tools used for image compression. 
+
+Plugins can be used to compress already uploaded images. Such plugins include [Imagify](https://imagify.io/wordpress/), [ShortPixel](https://wordpress.org/plugins/shortpixel-image-optimiser/), [Optimole](https://wordpress.org/plugins/optimole-wp/), and [WP Smush](https://wordpress.org/plugins/wp-smushit/). 
+
+Keep your image under 100 KB for the best performance.
+
+### Delete Unused Themes, Plugins, and Media Files
+
+This is a crucial website maintenance task. You should get rid of themes and plugins that are not in use because plugins and themes take space in the server so you're causing unnecessary strain and it's also a security risk. This involves deleting installed themes that are not in use and deleting deactivated plugins.
+
+### Use Well-optimized Themes
+
+Don't select some beautiful and impressive looking WordPress themes at the expense of performance. Pay special attention to theme optimization. Your WordPress theme should be elegant, lightweight, SEO friendly, fully responsive, and mobile optimized. 
+
+Some of the well-optimized WordPress themes are [Astra](https://wpastra.com/), [Udesign](https://dreamthemedesign.com/themes/u-design/), and [X Theme](https://theme.co/x).
+
+### Optimize WordPress Database
+
+Over time, the WordPress database will accumulate unnecessary data, clutter. This will slow down your website. The clutter could be post
+revisions, data that was created by already uninstalled plugins, and WordPress sessions. To optimize the database, you can use plugins. It is advisable to back up your database before trying to clean. 
+
+Some of the plugins available for database optimization are [WP-Optimize](https://wordpress.org/plugins/wp-optimize/) and [WP-Sweep](https://wordpress.org/plugins/wp-sweep/).
+
+![](wp-optimize-database-optimization.jpeg)
+
+See above screenshot of WP-Optimize 3.1.2 free version database optimization
+
+### Use Lazy Loading for Images and Videos
+
+Lazy loading is a web optimization technique where visible content is loaded first while content appearing below the fold is only downloaded and rendering when necessary. Instead of loading every image and video on a webpage, lazy loading downloads and render only what is visible to the user's screen.
+
+As the user scrolls down the page, more content is loaded. This significantly increases the loading speed of the content that is visible
+to the user. 
+
+You can activate lazy-loading using plugins such as [Lazy Load for Images](https://wordpress.org/plugins/lazy-load-for-images/) and [BJ Lazy Load](https://wordpress.org/plugins/bj-lazy-load/). 
```

Comment:
The BJ Lazy Load plugin hasn't been updated in 3 years or been tested in the last three major versions of WordPress so I would recommend removing it. Perhaps highlight the native lazy loading WordPress feature in WP 5.5? https://kinsta.com/blog/wordpress-5-5/#native-image-lazyloading-in-wordpress-core

Reply:
@louisefindlay23 This was a very helpful insight. I worked on it.